core/dbt/events/types.py

```diff
@@ -1721,6 +1721,14 @@ def cli_msg(self) -> str:
         return str(self.exc)
 
 
+@dataclass
+class IntegrationTestMessage(ShowException, ErrorLevel, CliEventABC):
+    msg: str
+
+    def cli_msg(self) -> str:
+        return self.msg
```

Comment:
On second thought, what do you think about doing something like this?
```suggestion
        return f"Integration Test: self.msg"
```